test/e2e/data/capi-operator/capv-provider.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: capv-system
+---
+apiVersion: operator.cluster.x-k8s.io/v1alpha1
```

Comment:
We use CAPIProvider manifests for the CAPI Operator resources management.

Reply:
Good point @Danil-Grigorev 👍